compiler/dialects/accfg.py

```diff
@@ -430,12 +430,29 @@ def launch_field_items(self) -> Iterable[tuple[str, IntegerAttr]]:
             yield name, val
 
 
+@irdl_op_definition
+class ResetOp(IRDLOperation):
+    name = "accfg.reset"
+
+    in_state = operand_def(StateType)
+
+    assembly_format = "$in_state attr-dict `:` type($in_state)"
+
+    def __init__(self, in_state: Operation | SSAValue):
+        super().__init__(operands=[in_state])
+
+    def get_modified_fields(self) -> set[str]:
+        # TODO: implement this part
+        return set()
```

Comment:
why is this here?

Reply:
good question! I originally wanted to have the lowering of the reset op in this PR as well, will remove!